packages/vuetify/src/components/VForm/VForm.ts

```diff
@@ -130,6 +131,12 @@ export default mixins(
   },
 
   render (h): VNode {
+    const children = this.$scopedSlots.default
```

Comment:
@KaelWD might have to correct me here, but I think this could be `this.$slots.default({ ... })`?

Reply:
If we do so ESlint throws TS2532

Reply:
Because it might not exist, you need to do `this.$scopedSlots?.default(...)`

Reply:
Since vue 2.6 all slots are now available on `$scopedSlots`

Reply:
This also needs to be added to `api-generator/maps/v-form.js`